### PR TITLE
misc/rawthrillspc.cpp: Add some notes about Doodle Jump hardware

### PR DESCRIPTION
**Pull request by clawgrip**

### CODE REVIEW

File: src/mame/misc/rawthrillspc.cpp [hunk 1]
```diff
@@ -119,12 +119,12 @@ ROM_END
 
 /*
  Doodle Jump Arcade, by default, uses a different PC than other Raw Thrills games:
-   -HP with an Asus M2N68-LA motherboard (socket AM3).
+   -HP with an Asus M2N68-LA motherboard (socket AM3, Fintek F8000 + Realtek RTL8201EL + Realtek ALC662 + nVidia NF-6100-430-N-A3 + ST L6740L).
```
Comment (angelosa) [inline src/mame/misc/rawthrillspc.cpp:122]: ... which runs over nVidia MCP61P chipset, which I *think* it's still nForce based?

Comment (angelosa) [reply src/mame/misc/rawthrillspc.cpp:122]: Also the official M2N68-LA codename is "Narra 3"

Comment (clawgrip) [reply src/mame/misc/rawthrillspc.cpp:122]: nVidia NF-6100-430-N-A3 is the integrated chip: GeForce 6100 + nForce 430

IMG_7713

Comment (clawgrip) [reply src/mame/misc/rawthrillspc.cpp:122]: What I'm trying to document with this PR are the chips and labels that can be seen or read on the actual PCBs, not the technical, marketing or commercial info about the components themselves.

Comment (angelosa) [reply src/mame/misc/rawthrillspc.cpp:122]: What I'm saying here is that yes, this makes this addition worth to stay in the same driver, but I'd like to have a note on top about being nForce chipset based (useful for later search).

Comment (clawgrip) [reply src/mame/misc/rawthrillspc.cpp]: Ah! Ok!

**Pull request merged**